installing/installing_bare_metal_single_node/install-sno-preparing-to-install-sno.adoc

```diff
@@ -0,0 +1,20 @@
+[id="preparing-to-install-sno"]
+= Preparing to install single node OpenShift (SNO)
+:context: install-sno-preparing
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+[IMPORTANT]
+====
+Single node OpenShift is a Developer Preview feature only. Developer Preview features
```

Comment:
"Installing {product-title} on a single node is a Developer Preview feature only."